### ISSUE TITLE
AssociatorInstances, not returning a result

### ISSUE THREAD

**Issue by tasleson** [runner-mei/gowbem#2]
When experimenting with AssociatorInstances, the XML returned from the provider has a single result, but the []CIMInstance is empty on return with no errors present.

At the moment I have no idea why this is happening and my limited golang skills are hindering the debug process :-)

**Comment by tasleson** [comment]
OK, looking at the XML parser and how it appears to work, it looks like the XML has `VALUE.OBJECTWITHPATH` instead of the expected `INSTANCE` in the results.  This seems to address the issue, but not %100 sure of the solution presented here and I haven't validated the resultant instance returned.  Would want to do more testing with more result returned and get a confirmation from others before doing a pull request on this.

```go
diff --git a/client_cimxml.go b/client_cimxml.go
index ef1d8c2..e5636c0 100644
--- a/client_cimxml.go
+++ b/client_cimxml.go
@@ -905,9 +905,9 @@ func (c *ClientCIMXML) AssociatorInstances(namespaceName string, instanceName CI
                return nil, err
        }
 
-       results := make([]CIMInstance, len(resp.Message.SimpleRsp.IMethodResponse.ReturnValue.Instances))
-       for idx, _ := range resp.Message.SimpleRsp.IMethodResponse.ReturnValue.Instances {
-               results[idx] = &resp.Message.SimpleRsp.IMethodResponse.ReturnValue.Instances[idx]
+       results := make([]CIMInstance, len(resp.Message.SimpleRsp.IMethodResponse.ReturnValue.ValueObjectWithPaths))
+       for idx, _ := range resp.Message.SimpleRsp.IMethodResponse.ReturnValue.ValueObjectWithPaths {
+               results[idx] = resp.Message.SimpleRsp.IMethodResponse.ReturnValue.ValueObjectWithPaths[idx].Instance
        }
        return results, nil
 }

```

**Comment by tasleson** [comment]
This appears to be corrected with 47edfadf9bcb8ce04233a4cd741d70dea73a3390, thanks!
